python/lsst/obs/base/makeRawVisitInfo.py

```diff
@@ -241,6 +241,15 @@ def popMjdDate(self, md, key, timesys=None):
         return BadDate
 
     @staticmethod
+    def eraFromLstAndLongitude(lst, longitude):
```

Comment:
I've documented longitude and added a test to check that it raises on float-Angle.